### ISSUE TITLE
[clsim] Overview diagram structure (Trac #2071)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2063]
Clsim consists of many files, and it is difficult to find relations between files. It is also not clear which files are used for GPU code and CPU code. A diagram to show the clsim structure will help for users and developers.

Migrated from https://code.icecube.wisc.edu/ticket/2071
```json
{
    "status": "closed", 
    "changetime": "2019-03-22T13:59:39", 
    "description": "Clsim consists of many files, and it is difficult to find relations between files. It is also not clear which files are used for GPU code and CPU code. A diagram to show the clsim structure will help for users and developers.", 
    "reporter": "mase", 
    "cc": "mase, summer.blot, jmicallef, claudio.kopper", 
    "resolution": "fixed", 
    "_ts": "1553263179201122", 
    "component": "combo simulation", 
    "summary": "[clsim] Overview diagram structure", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-08-03T21:17:25", 
    "milestone": "Vernal Equinox 2019", 
    "owner": "amedina", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170803T21:22:03: mase changed milestone from "" to "Long-Term Future"

**Comment by icecube-trac** [comment]
Trac update at 20180523T18:38:28: 
* olivas changed owner from "" to "olivas"
* olivas changed milestone from "Long-Term Future" to "Autumnal Equinox 2018"
* olivas changed status from "new" to "assigned"
* olivas commented:

> Going to see if we can't get a sub-team of the Strike Team to tackle this over the next few months.

**Comment by icecube-trac** [comment]
Trac update at 20180922T10:33:27: olivas changed milestone from "Autumnal Equinox 2018" to "Winter Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20180922T10:46:16: olivas changed summary from "Overview diagram of clsim structure" to "[clsim] Overview diagram structure"

**Comment by icecube-trac** [comment]
Trac update at 20181114T14:44:25: olivas changed owner from "olivas" to ""

**Comment by icecube-trac** [comment]
Trac update at 20181127T02:37:31: olivas commented:

> https://icecube-spno.slack.com/archives/simulation/p1452528445000297 (from ticket #1496)

**Comment by icecube-trac** [comment]
Trac update at 20181207T17:12:04: olivas changed owner from "" to "amedina"

**Comment by icecube-trac** [comment]
Trac update at 20190311T14:49:41: 
* olivas changed cc from "mase, summer.blot, cweaver, dima" to "mase, summer.blot, jmicallef, claudio.kopper"
* olivas commented:

> Jessie agreed to help with this ticket.

**Comment by icecube-trac** [comment]
Trac update at 20190318T16:18:46: amedina changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20190322T13:59:39: 
* olivas commented:

> In changeset 172269:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="172269"
> just a few, minor tweaks.  thanks to jessie, andres, and claudio for updating those docs.  closes #2063.
> ```


* olivas changed status from "accepted" to "closed"
* olivas changed resolution from "" to "fixed"
